modules/friendlytag.js

```diff
@@ -207,6 +207,13 @@ Twinkle.tag.callback = function friendlytagCallback() {
 
 	if (Twinkle.tag.mode === "article") {
 
+		result.quickfilter.autocomplete = 'off'; // disable browser suggestions for this field
+		result.quickfilter.addEventListener('keypress', function(e) {
+			if (e.keyCode === 13) { // prevent enter key from submitting
+				e.preventDefault();
+				return false;
```

Comment:
This couldn't have been added to the main event function because that's a `keyup` handler, and the form would have already been submitted by the time of `keyup`.